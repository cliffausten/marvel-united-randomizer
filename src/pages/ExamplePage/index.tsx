import React from 'react';
import ExampleComponent from 'components/ExampleComponent';
import styles from './ExamplePage.module.css';

const ExamplePage: React.FC = () => {
  return (
    <div>
      <div className={styles['basket-header']}>
        <h1>Example Page</h1>
      </div>
      <div className={styles['basket-grid']}>
        <ExampleComponent />
      </div>
    </div>
  );
};

export default ExamplePage;
