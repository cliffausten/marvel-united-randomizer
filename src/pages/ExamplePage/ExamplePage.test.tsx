import React from 'react';
import ExamplePage from './index';
import { shallow } from 'enzyme';

describe('ExamplePage', () => {
  it('Render', () => {
    const examplePage = shallow(<ExamplePage />);

    expect(examplePage).toMatchSnapshot();
  });
});
