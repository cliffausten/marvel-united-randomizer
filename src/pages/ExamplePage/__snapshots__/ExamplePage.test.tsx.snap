// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExamplePage Render 1`] = `
<div>
  <div
    className="basket-header"
  >
    <h1>
      Example Page
    </h1>
  </div>
  <div
    className="basket-grid"
  >
    <Memo(wrappedComponent) />
  </div>
</div>
`;
